test/MainEvent.cxx

```diff
@@ -131,9 +131,12 @@ int main(int argc, char **argv)
    Int_t branchStyle = 1; //new style by default
    if (split < 0) {branchStyle = 0; split = -1-split;}
 
+   ROOT::EnableImplicitMT(4);
```

Comment:
This should be optional (so we need to add a command line argument) and default to not be enabled (as is the test now complain if ROOT was not built with IMT enabled).